pennylane_lightning/lightning_tensor/_tensornet.py

```diff
@@ -223,20 +221,35 @@ def _preprocess_state_vector(self, state, device_wires):
         if len(device_wires) == self._num_wires and Wires(sorted(device_wires)) == device_wires:
             return np.reshape(state, output_shape).ravel(order="C")
 
-        # generate basis states on subset of qubits via the cartesian product
-        basis_states = np.array(list(product([0, 1], repeat=len(device_wires))))
+        local_dev_wires = np.array(device_wires.tolist())[::-1]
```

Comment:
Why is this cast to a numpy array?
Is it preferable to do so, rather than use native Python?

Reply:
I was looking to avoid modifying the list `device_wires` by copying the data. Do you suggest to use `list`?

Reply:
I think it is often better to stick to native types, unless there is a reason to use numpy here of course. If there's no appreciable runtime difference between the two, I'd use the non-numpy option. If numpy gives a performance benefit (assuming we are setting the dtype to int), then fine to use it with that reason in mind.

Reply:
I changed to Python `List` :zap: